audio-worklet/basic/event-processor-state.html

```diff
@@ -23,6 +23,16 @@ <h1>
       <p>
         <strong>NOTE</strong>: Open up the console to see the events being
         fired.
+        <ul>
+          <li>- An instance of <code>TimerProcessor</code> will be in "pending"
```

Comment:
What's the "-" for?  Shouldn't the li tag take care of that?

Reply:
Unfortunately CSS set up removes the bullet from `<li>`. Is this blocking issue for this PR? Then I can rewrite CSS to avoid it, but not in this PR.

Reply:
No, it's not a blocker, but it seems really odd to make CSS li tag have no bullet, and then put a - bullet in for li tags here.

Why do we need a custom style sheet here anyway?